src/TTCache.php

```diff
@@ -31,20 +32,26 @@ class TTCache
 
     protected TaggedStore $cache;
 
-    public function __construct(CacheInterface $cache, Closure $keyHasher = null)
+    /**
+     * @var string
+     */
+    private string $specialKeyDelimeter;
+
+    public function __construct(CacheInterface $cache, Closure $keyHasher = null, string $specialKeyDelimeter = ':')
     {
-        $this->cache = new TaggedStore($cache);
+        $this->cache = new TaggedStore($cache, $specialKeyDelimeter);
         $this->keyHasher = $keyHasher ?? fn ($x) => $x;
```

Comment:
Should we make the default keyHasher `md5` / `sha1` / ... instead? Now realize it would be better for this to have sand production defaults instead of easy debugging ones.
If we set a proper hash as the default and `-` as the separator, then we don't need the special constructors at all.